packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -5674,6 +5674,24 @@ void main() {
         }),
       );
     });
+
+    testWidgets('TextEditingController.buildTextSpan receives build context', (WidgetTester tester) async {
+      final AccentColorTextEditingController controller = AccentColorTextEditingController('a');
+      await tester.pumpWidget(MaterialApp(
+        theme: ThemeData.light().copyWith(accentColor: const Color.fromARGB(255, 1, 2, 3)),
+        home: EditableText(
+          controller: controller,
+          focusNode: FocusNode(),
+          style: Typography.material2018(platform: TargetPlatform.android).black.subtitle1!,
+          cursorColor: Colors.blue,
+          backgroundCursorColor: Colors.grey,
+        ),
+      ));
+
+      final RenderEditable renderEditable = findRenderEditable(tester);
+      final TextSpan textSpan = renderEditable.text!;
+      expect(textSpan.style!.color, const Color.fromARGB(255, 1, 2, 3));
```

Comment:
Nit: Maybe declare this color as a variable so that it's obvious where it comes from.  Up to you though.